region/wreckedship/main.json

```diff
@@ -2251,6 +2251,32 @@
         {
           "from": 3,
           "to": [
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "X-Ray Climb",
+                  "notable": false,
+                  "requires": [
+                    "canRightFacingDoorXRayClimb",
+                    {"adjacentRunway": {
+                      "fromNode": 3,
+                      "usedTiles": 0.5,
+                      "overrideRunwayRequirements": true,
+                      "useFrames": 200
+                    }},
+                    {"or": [
+                      {"spikeHits": 3},
+                      {"and": [
+                        {"not": "f_DefeatedPhantoon"},
+                        "canRiskPermanentLossOfAccess"
+                      ]}
+                    ]}
+                  ],
+                  "devNote": "FIXME: See if refreshing I-frames using XRay can reduce the required number of hits."
```

Comment:
Refresh Iframes does not work here.  I've only seen it work on the etecoon spikes and enemy projectiles.  So maybe you need airspikes.
However, Xray does make it easier to take extra damage since your iframes will run out while using it.  Once you reach the spikes its a little hard to know how to handle it.  You can't fall out, and jumping the rest of the way is never safe.

Reply:
Good to know. I've added some more detail to the note accordingly